src/main/java/com/bytezone/dm3270/display/Field.java

```diff
@@ -184,6 +185,26 @@ public String toString() {
         startFieldAttribute.getAcronym(), getText());
   }
 
+  @Override
+  public boolean equals(Object o) {
```

Comment:
You need to also compare the text. And doesn't make sense to compare isVisible since acronym already considers the visibility